get-started-with-angular/notes/01-say-hello-world-to-angular-2.md

```diff
@@ -0,0 +1,82 @@
+# Say "Hello World" to Angular 2
+
+**[📹 Video](https://egghead.io/lessons/angular-say-"hello-world"-to-angular-2)**
+
+## Setting up with the Angular CLI ⚡
+
+To get started with our Angular application, we'll install the [Angular CLI](https://cli.angular.io/).
+
+
+In the terminal:
+```bash
+npm install -g @angular/cli
+```
+Even if you've installed the Angular CLI in the past, running this command will update it to the latest version. 
+
+Now, change directory to where you'd like your project folder to live. Then, in the terminal, we create our application's project folder:
+```bash
+ng new angular2-fundamentals
+```
+In the terminal, you will receive two prompts:
+```bash
+? Would you like to add Angular routing?
+? Which stylesheet format would you like to use?
+```
+For the first, enter "n" for no. For the second, select CSS.
+
+After a few moments we should now have a folder in the current directory with the name **angular2-fundamentals**
+
+Change directories into your new project folder:
+```bash
+cd angular2-fundamentals
+```
+And start your development server:
+```bash
+ng serve
+```
+Once loaded, we can navigate to **localhost:4200** in our browser.
+
+If you're following along with the lesson and using the latest version of Angular CLI, you may be surprised to see a much more elaborate page compared to the one in the lesson:
+
+![Angular CLI Default Application Page](.\images\1.png)
```

Comment:
This image link is broken. Also when naming the pictures, the way we need them named is `{lesson-slug}+{short-picture-description}`. The reason is when we upload our pictures to cloudinary, it makes it so that we can search for pictures for a certain course/lesson. I'll need you to fix that before we can get these uploaded.